language/resource-api/README.md

```diff
@@ -432,23 +432,26 @@ Password attributes should also set `sensitive: true` to ensure that the data is
 
 #### Schema
 
-A transport schema accepts the following keywords:
+To align with [Bolt's inventory file](https://puppet.com/docs/bolt/latest/inventory_file.html), a transport schema prefers the following keywords (when relevant):
 
-* `uri`: use when you need to specify a specific URL to connect to. All of the following keys will be computed from the `uri` when possible. In the future more url parts may be computed from the URI.
-* `host`: use to specify an IP or address to connect to.
+* `uri`: use when you need to specify a specific URL to connect to. Bolt will compute the following keys from the `uri` when possible. In the future more url parts may be computed from the URI.
 * `protocol`: use to specify which protocol the transport should use for example `http`, `https`, `ssh` or `tcp`.
-* `user`: the user the transport should connect as.
+* `host`: use to specify an IP or address to connect to.
 * `port`: the port the transport should connect to.
+* `user`: the user the transport should connect as.
+* `password`: the password for the specified user.
 
 Do not use the following keywords when writing a schema:
 
+* `implementations`: reserved by Bolt.
 * `name`: transports should use `uri` instead of name.
 * `path`: reserved as a uri part.
 * `query`: reserved as a uri part.
-* `run-on`: This is used by Bolt to determine which target to proxy to. Transports should not rely on this key.
-* `remote-transport`: This is used to determine which transport to load. It should always be the transport class named "declassified".
 * `remote-*`: any key starting with `remote-` is reserved for future use.
-* `implementations`: reserved by Bolt.
+* `remote-transport`: determines which transport to load. It is always the transport class named "declassified".
```

Comment:
> It is always the transport class named "declassified".

The sentence doesn't make sense to me.  It reads like `remote-transport: declassified`. 

Should it be: It is always the transport class name "declassified".` 

If so, I believe it may make more sense to be written as:
It is always the transport class name written in lowercase. e.g. `class Panos` becomes `panos`

Reply:
It should always be `the transport name`. The class name for the transport is the snake case (`SnakeCase`) version of the transport name.

good spot, @da-ar